spec.html

```diff
@@ -27824,20 +27824,20 @@ <h1>Math.floor ( _x_ )</h1>
         <h1>Math.fround ( _x_ )</h1>
         <p>When the `Math.fround` method is called with argument _x_, the following steps are taken:</p>
         <emu-alg>
-          1. If _x_ is *NaN*, return *NaN*.
-          1. If _x_ is one of *+0*, *-0*, *+&infin;*, *-&infin;*, return _x_.
-          1. Let _x32_ be the result of converting _x_ to a value in IEEE 754-2019 binary32 format using roundTiesToEven mode.
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, return *NaN*.
+          1. If _n_ is one of *+0*, *-0*, *+&infin;*, or *-&infin;*, return _n_.
+          1. Let _x32_ be the result of converting _n_ to a value in IEEE 754-2019 binary32 format using roundTiesToEven mode.
```

Comment:
We should rename `x32` and `x64` to `n32` and `n64` respectively.

Reply:
Yeah, I was on the fence about that, left it out to minimize diff. I'll put it in.